### DISCUSSION TITLE
Window moving and resizing shortcuts

### DISCUSSION THREAD

**Discussion by mrcnk** [MatthiasGrandl/Loungy#38] [Ideas]
Hey, first of all, great app and experience so far. I love Zed, so of course it's easy to love Loungy.

My idea is to add shortcuts to allow window management like Raycast (lh - Left Half, rh - Right Half, tlq - Top Left Quarter etc). There's a new window tiling in MacOS Sequoia, so I guess the window management can just trigger the new hotkeys (see Window -> Move & Resize) on MacOS.

**Comment by MatthiasGrandl** [comment]
@mrcnk thanks for your interest in Loungy. Sadly this is definitely out-of-scope for this project. I use a tiling window manager, so the Raycast window management was never of interest for me. It's also something that is probably impossible to implement in a cross-platform manner.

**Comment by mrcnk** [comment]
@MatthiasGrandl I'm cool with that, but can we have an extension interface (can be as simple as writing Lua scripts in Hammerspoon) in Loungy?

**Reply by MatthiasGrandl** [reply]
well if I ever get around to finishing the plugin system you can run anything. in Raycast I once wrote a plugin to interact with Hammerspoon IPC to get around some Raycast limitations. Sadly you won't be able to interact with native APIs (like MacOS accessibility) unless explicitely exposed by the Loungy extension host.
